client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -545,6 +554,20 @@ class DeltaSharingRestClient(
     }
   }
 
+  private def getEncodedMetadataParams(
```

Comment:
Consider documenting this function.
An example encoded string will be useful to visualize what is happening.

Reply:
added.